app/javascript/article-form/articleForm.jsx

```diff
@@ -395,12 +383,13 @@ export default class ArticleForm extends Component {
       let moreConfigBottomButton = '';
       if (version === 'v2') {
         moreConfigBottomButton = (
-          setupImageButton({
-            className: 'articleform__detailsButton articleform__detailsButton--moreconfig articleform__detailsButton--bottom',
-            imgSrc: ThreeDotsIcon,
-            imgAltText: 'menu dots',
-            onClickCallback: this.toggleMoreConfig
-          })
+          <button
```

Comment:
is there a reason why this refactoring was undone? It was merged literally 6 days ago eheh https://github.com/thepracticaldev/dev.to/pull/4159

Reply:
It wasn't me. It's done by Prettier. The same goes for the other two cases.

Reply:
this undoing of a PR is weird, we should look into how prettier is configured, mmmh

Reply:
No worries. I'll put it back. This is why code climate is complaining anyway so...

Reply:
I found a better way that doesn't force Prettier to re-write the code and code climate is fine with that also. `setupImageButton` is a function that returns with a component so it's basically a functional component so we should declare and use it according to the convention by the docs.

Reply:
Great job!!